src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifestBase.cs

```diff
@@ -426,6 +426,26 @@ public void CheckForStableAssetsInNonIsolatedFeeds()
             HashSet<TargetFeedConfig> feedConfigsForSymbols = FeedConfigs[symbolCategory];
             Dictionary<string, string> serversToPublish =
                 GetTargetSymbolServers(feedConfigsForSymbols, msdlToken, symWebToken);
+            HashSet<string> excludeFiles = new HashSet<string>();
+            
+            if(File.Exists(symbolPublishingExclusionsFile))
+            {
+                Log.LogMessage(MessageImportance.High, $"SymbolPublishingExclusionFile exists");
+                string[] files = File.ReadAllLines(symbolPublishingExclusionsFile);
+
+                foreach(var file in files)
+                {
+                    if(!string.IsNullOrEmpty(file))
+                    {
+                        Log.LogMessage(MessageImportance.High, $"Exclude the file {file} from publishing to symbol server");
+                        excludeFiles.Add(file);
+                    }
+                }
+            }
+            else
+            {
+                Log.LogMessage(MessageImportance.High, $"SymbolPublishingExclusionFile does not exits, check path ${symbolPublishingExclusionsFile} ");
```

Comment:
[wording nit]

```suggestion
                Log.LogMessage(MessageImportance.High, $"SymbolPublishingExclusionFile was not found at ${symbolPublishingExclusionsFile} ");
```

If you don't want to change the wording, there's a typo, exits -> exists.

Reply:
changed the wording as suggested :)